### PR TITLE
Added blacklist and minimum liquidity threshold configurations for pricing

### PR DESCRIPTION
**Pull request by steegecs**

### CODE REVIEW

File: subgraphs/uniswap-forks/protocols/quickswap/config/networks/matic/matic.ts [hunk 1]
```diff
@@ -81,4 +81,10 @@ export class QuickswapMaticConfigurations implements Configurations {
   getUntrackedPairs(): string[] {
     return toLowerCaseList([]);
   }
+  getUntrackedTokens(): string[] {
+    return [];
+  }
+  getMinimumLiquidityThreshold(): BigDecimal {
+    return BigDecimal.fromString("100000");
```
Comment (this-username-is-taken) [inline subgraphs/uniswap-forks/protocols/quickswap/config/networks/matic/matic.ts:88]: Might be better if we have this as a constant in a shared file?

Comment (steegecs) [reply subgraphs/uniswap-forks/protocols/quickswap/config/networks/matic/matic.ts:88]: Yeah - I can put the value in the protocol specific constants folder. Then if more flexibility is wanted, I could add constants.ts to the particular network folder.

Comment (steegecs) [reply subgraphs/uniswap-forks/protocols/quickswap/config/networks/matic/matic.ts]: Added constants for 100,000 and 400,000 into the shared constants file.

**Comment by this-username-is-taken** [comment]
Great work! Appreciate all the debugging!

**Pull request merged**